nilearn/connectome/tests/test_connectivity_matrices.py

```diff
@@ -23,6 +23,18 @@
 from sklearn.covariance import EmpiricalCovariance, LedoitWolf
 from sklearn.utils import check_random_state
 
+CONNECTIVITY_KINDS = [
```

Comment:
I'd expect a tuple for this (unless there is a reason I didn't see for using a list):

```suggestion
CONNECTIVITY_KINDS = (
```

Reply:
That is what I understand also, hence relying on a tuple rather than a list.